lib/src/functions/color.dart

```diff
@@ -475,7 +475,10 @@ final module = BuiltInModule("color", functions: <Callable>[
 
   _function("channel", r"$color, $channel, $space: null", (arguments) {
     var color = _colorInSpace(arguments[0], arguments[2]);
-    var channelName = arguments[1].assertString("channel").text.toLowerCase();
+    var channelName = (arguments[1].assertString("channel")
```

Comment:
Nit: This pattern is used in one other place besides these two new uses (although the other one doesn't transform the string to lowercase). This can be extracted to a helper function, which may be useful if the same has to be repeated more times 🤔